### PR TITLE
docs: Update the OCM walkthrough for clarity

### PR DESCRIPTION
**Pull request by jasonmadigan**
Some edits to add extra clarity to the OCM PoC setup guide, including:

- Refs to which terminal to use to run various setup commands
- Markdown syntax highlighting for code blocks

I'll hold off on a merge until https://github.com/Kuadrant/multicluster-gateway-controller/pull/232 lands, as that known issue which I've removed a ref to won't be fixed until that is merged.

### CODE REVIEW

File: docs/ocm-control-plane-walkthrough.md [hunk 1]
```diff
@@ -108,39 +116,39 @@ EOF
...
 
 ### Check the managed zone
 
-Lets ensure our managedzone is present. Open a new terminal: 
+Lets ensure our `managedzone` is present. In `T1`, run the following:
 
-```
+```bash
 export KUBECONFIG=$(pwd)/local/kube/control-plane.yaml
 kubectl get managedzone -n multi-cluster-gateways
```
Comment (david-martin) [inline docs/ocm-control-plane-walkthrough.md:139]: Small suggestion here for something that is a little annoying.
Split the cmd and the output into separate blocks.

File: docs/ocm-control-plane-walkthrough.md [hunk 2]
```diff
@@ -108,39 +116,39 @@ EOF
...
 
 NAME          DOMAIN NAME      ID                                  RECORD COUNT   NAMESERVERS                                                                                        READY
 mgc-dev-mz   replace.this   /hostedzone/Z08224701SVEG4XHW89W0   7              ["ns-1414.awsdns-48.org","ns-1623.awsdns-10.co.uk","ns-684.awsdns-21.net","ns-80.awsdns-10.com"]   True
-
 ```
 
 ### Create a gateway
 
-We know will create a multi-cluster gateway definition in the hub cluster. Make sure to replace the sub.replace.this with a subdomain of your root domain.
+We know will create a multi-cluster gateway definition in the hub cluster. In `T1`, run the following:
```
Comment (david-martin) [inline docs/ocm-control-plane-walkthrough.md:147]:
```suggestion
We will now create a multi-cluster gateway definition in the hub cluster. In `T1`, run the following: 
```
?

File: docs/ocm-control-plane-walkthrough.md [hunk 3]
```diff
@@ -166,16 +174,16 @@ In the hub cluster there will be a single gateway definition but no actual gatew
...
 
 ```
 
-Now on the hub cluster you should find there is a configured gateway and instantiated gateway
+Now on the hub cluster you should find there is a configured gateway and instantiated gateway. In `T1`, run:
 
-```
+```bash
 kubectl get gateway -A
 kuadrant-multi-cluster-gateways   prod-web   istio                                         172.32.200.0                29s
```
[the remark this answers is not in the document]
Comment (david-martin) [reply docs/ocm-control-plane-walkthrough.md:188]: There's at least 1 other instance of this further on

File: docs/ocm-control-plane-walkthrough.md [hunk 4]
```diff
@@ -268,55 +275,55 @@ EOF
...
-```
+```bash
 kubectl get dnsrecord -n multi-cluster-gateways -o=yaml
 ```
 
 Give DNS a minute or two to update. You should then be able to execute
 
-```
+```bash
 dig sub.replace.this
```
Comment (david-martin) [inline docs/ocm-control-plane-walkthrough.md:296]: To make things a little smoother, what about introducing a env var early on in the walkthrough for the hostname?

File: docs/ocm-control-plane-walkthrough.md [hunk 5]
```diff
@@ -268,55 +275,55 @@ EOF
...
 kubectl label managedcluster kind-mgc-workload-1 ingress-cluster=true
-
 ```
 
-This has added our workload-1 cluster to the ingress clusterset. Next we need to modify our placement
+This has added our workload-1 cluster to the ingress clusterset. Next we need to modify our placement. In `T1`, run:
 
-```
+```bash
 kubectl edit placement http-gateway -n multi-cluster-gateways
```
Comment (david-martin) [inline docs/ocm-control-plane-walkthrough.md:319]: a patch cmd would be nice here

File: docs/ocm-control-plane-walkthrough.md (hunk 1 above)
Comment (jasonmadigan) [reply docs/ocm-control-plane-walkthrough.md:139]: Hard agree - I'll do these here and for the others.

File: docs/ocm-control-plane-walkthrough.md (hunk 4 above)
Comment (jasonmadigan) [reply docs/ocm-control-plane-walkthrough.md:296]: Had a similar thought when I last went through this on Friday

**Comment by jasonmadigan** [comment]
@david-martin couple of changes to fixup those comments.

**Review by david-martin — APPROVED** [review]

**Pull request merged**